.github/workflows/preview.yml

```diff
@@ -0,0 +1,44 @@
+name: PR preview release
+
+on:
+  pull_request:
+    paths-ignore:
+      - ".github/workflows/test.yml"
+      - ".github/workflows/codeql.yml"
+      - ".github/workflows/coverage.yml"
+      - ".github/workflows/database-tests-compose.yml"
+      - ".github/workflows/database-tests-windows.yml"
+      - ".github/workflows/database-tests.yml"
+      - ".github/workflows/publish-package.yml"
+      - "docs/**"
+      - "test/**"
+      - "resources/**"
+      - ".gitignore"
+      - ".prettierignore"
+      - ".prettierrc.js"
+      - "eslint.config.mjs"
+      - "*.md"
+      - "LICENSE"
+
+jobs:
+  release:
+    name: Release preview build
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+        with:
+          cache: "npm"
+          node-version: "20.x"
```

Comment:
use `20` instead of `"20.x"` for consistency

Reply:
in the [publish workflow[https://github.com/typeorm/typeorm/blob/013fb4e8d5e4f6fb35a939c57767990011bcb6c3/.github/workflows/publish-package.yml) we're using this syntax, should I also align that file in this PR?